app/models/service_template_ansible_playbook.rb

```diff
@@ -208,6 +210,15 @@ def self.new_job_template_required?(info, action, service_template)
   end
   private_class_method :new_job_template_required?
 
+  def rollback_job_templates
+    auth_user = User.current_userid || 'system'
+    config_info = options[:config_info]
+    config_info.keys.each do |action|
+      ManageIQ::Providers::EmbeddedAnsible::AutomationManager::ConfigurationScript
+        .delete_in_provider_queue(options[:manager_id], { :name => self.class.send(:build_name, name, action) }, auth_user)
```

Comment:
If this `job_template` was built before this fix was in place, there would be no `manager_id` saved.  I'm thinking I could add something like 
```
options[:manager_id] || ManageIQ::Providers::EmbeddedAnsible::AutomationManager.last
```